### ISSUE TITLE
Infinite Scrolling example with the latest version

### ISSUE THREAD

**Issue by davidpelayo** [idiotWu/smooth-scrollbar#251]
Where can I find an example working with some infinite scrolling?
I now from the [react-smooth-scrollbar](http://idiotwu.github.io/react-smooth-scrollbar/) of an example working with the version 7, but I can't find it with the latest version.

All what I find is the migration guide that says that _Reason: can be implemented with ScrollbarPlugin_.

However, the infinite scrolling is still a common use case for data pagination and I can't find any example on how to use it with this tool.

Thanks in advance.

**Comment by idiotWu** [comment]
You can use the exactly same code from 7.x version if you don't want to use `ScrollbarPlugin` :).

**Comment by davidpelayo** [comment]
That's right. But I was curious of why and how the migration guide says it could be done using the `ScrollbarPlugin` but there is no example of it anywhere. Wouldn't it be then needed either upgrading the examples or simply clarifying that with the latest version the consumers could still implement the code from the 7.x version and that will work correctly?
